src/common/components/Buttons/Button.tsx

```diff
@@ -31,6 +35,9 @@ const Button: React.FC<TButton> = ({
   const classes = cn(
     {
       'button inline-flex items-center px-2 h-10 justify-center rounded-2xl transition duration-300 focus:outline-none active:shadow-none focus:shadow-btnOutline max-w-full leading-none': true,
+      'w-full': fluid,
+      'w-60': !fluid && !width,
```

Comment:
Yes, it can be changed by the className. Generally the buttons are w-60 or w-full. If you need the custom size, you can change it. I added this feature for code optimization
className='!w-32' (important)

Reply:
Please, just remove fluid and w-60
Fluid because w-full class is a better way
w-60 because I searched design for w-60, which is 240px, and didn't find any 240px button. I agree then you might find such button in design occasionally, but it should not be a default.

For such common component as Button, better have no default width at all, please

Reply:
And after removal no longer need for !important